lib/rswag/specs/example_helpers.rb

```diff
@@ -2,24 +2,56 @@
 
 require "rswag/specs/request_factory"
 require "rswag/specs/response_validator"
+require "faraday"
 
 module Rswag
   module Specs
     module ExampleHelpers
       def submit_request(metadata)
         request = RequestFactory.new.build_request(metadata, self)
 
-        send(
-          request[:verb],
-          request[:path],
-          request[:payload],
-          request[:headers]
-        )
+        if request[:external_api]
+          send_external_request(request)
+        else
+          send(
+            request[:verb],
+            request[:path],
+            request[:payload],
+            request[:headers]
+          )
+        end
       end
 
       def assert_response_matches_metadata(metadata)
         ResponseValidator.new.validate!(metadata, last_response)
       end
+
+      private
+
+      def send_external_request(request)
+        connection = Faraday.new(
+          url: request[:path],
+          params: request[:payload],
+          headers: request[:headers]
+        )
+        
+        choise_faraday_method(request[:verb], connection)
```

Comment:
better to use more generatl `run_request` https://www.rubydoc.info/github/lostisland/faraday/Faraday/Connection#run_request-instance_method